modules/bankfeed/api/bankfeed.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import { getQtyFromRequest } from '../../../utils/route-utils';
+
+import BankData from '../data/bankdata'; //Bank feed json
+
+module.exports = function (app: core.Express) {
+    /**
+     * @openapi
+     * '/bank/feed/:qty':
+     *   get:
+     *     tags:
+     *     - Bankfeed
```

Comment:
@siddharth1729 Seems like we are getting multiple tags related to "Bankfeeds"/"Bankfeed".